record/record.controller.js

```diff
@@ -195,6 +200,16 @@
             return ref.canCreate;
         };
 
+        vm.canDeleteRelated = function(relatedRef) {
```

Comment:
We talk about this and you might have changed it already, but I'm going to comment for the future reference. I think we should use the same terminology at least in the code. If it's called `unlink` somewhere else, we should keep using that name. And if you want to change it, we should be consistent and change the other parts of the code.